setup.py

```diff
@@ -45,6 +44,7 @@
     maintainer=wixpy.MAINTAINER,
     maintainer_email=wixpy.MAINTAINER_EMAIL,
     license=wixpy.LICENSE,
+    install_requires=['distro'],
```

Comment:
```suggestion
    install_requires=['distro'],
    setup_requires=['distro'],
```
Required by new `dependencies.py`, which is imported in `setup.py`.